ansible/roles/ocp4-workload-app-deploy-homework/templates/cluster_resource_quota.j2

```diff
@@ -1,7 +1,7 @@
 apiVersion: quota.openshift.io/v1
 kind: ClusterResourceQuota
 metadata:
-  name: "clusterquota-{{ ocp_username }}-{{ guid }}"
```

Comment:
You probably want to keep the old way too, and make sure it's absent. (Or cleanup quotas manually).

Is it the only resource having the username in its name?

Reply:
If I do the remove will fail...